### ISSUE TITLE
Meetup in Delhi , India ?

### ISSUE THREAD

**Issue by evalsocket** [ipfs/community#280]
I'm currently in Delhi and would be interested in getting an IPFS community going. if anyone is interested in joining. I could also use some help in organizing it and getting a place to host it.

**Comment by atvanguard** [comment]
Wow, this is brilliant. Though needless to say, you would be able to find some audience in Bangalore; not sure about Delhi

**Comment by HariharasudhanAS** [comment]
I would love to collaborate. Feel free to mail me at harimailbox99 (at) Gmail.

**Comment by KottackalJacob** [comment]
Could you setup the meetup in South India? Bangalore or Trivandrum?

India is a big country and traveling from South to North (for our team) might not be ideal. If in Trivandrum, we could get the space and get some interested folks.

We have and are doing some POCs using IPFS. So this is great news.

Feel free to contact me at jacobninan(at)gmail.com.

**Comment by atvanguard** [comment]
@HariharasudhanAS @KottackalJacob Does Bangalore suit you guys?
We at [Semantics3](https://www.semantics3.com/) are doing a project on IPFS. If Bangalore works, we can do this asap.

**Comment by diasdavid** [comment]
Let me know how we can help :) I can create the meetup.com group for Bangalore and send you some swag :)

**Comment by evalsocket** [comment]
Can we plan it remotely on zoom and just share experiences. We can cover Delhi,Mumbai and Bangalore.

**Comment by KottackalJacob** [comment]
@atvanguard works for us.

**Comment by HariharasudhanAS** [comment]
@atvanguard Can finalize Bangalore I guess.

**Comment by KottackalJacob** [comment]
@diasdavid Any plans for the IPFS core developers to attend meetups in India? We need some support / hand holding if possible. I feel that we are not using even 10 % power of IPFS and related tools.

**Comment by HariharasudhanAS** [comment]
@KottackalJacob I agree. Developers are abundant here in India, but the penetration of IPFS is really low.

**Comment by evalsocket** [comment]
Can we plan online meetup on zoom ?  @HariharasudhanAS @atvanguard @diasdavid @KottackalJacob 
 Suggest time for monthly meetup (Online)

**Comment by rkapurbh** [comment]
Hey @evalsocket - is there an email I can reach you at? I'm working on a crypto project and I'm looking for fellow Indian collaborators to work w/ me - I would love to discuss further on an email thread!

**Comment by syedraj2** [comment]
Hi guys I'm having alots of query related to ipfs I need someone support to implement can anyone of u share there knowledge it would be helpful for me.

**Comment by evalsocket** [comment]
@syedraj2 check issue [#1633](https://github.com/ipfs/go-ipfs/issues/1633) 
[documentation](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#private-networks)
and [quick Intro](https://medium.com/coinmonks/setting-up-your-first-distributed-private-storage-network-on-ipfs-part-1-a6ff15222b90)

**Comment by syedraj2** [comment]
I tried this too... But I don't know how to create use swarm key to publish the private network

**Comment by kishansagathiya** [comment]
@daviddias @evalsocket 
I live in Bangalore. I would to help with this.

**Comment by evalsocket** [comment]
@kishansagathiya Contact me evalsocket@protonmail.com

**Comment by evalsocket** [comment]
IPFS 1st Meetup Bangalore
Date & Time :  13th Jan at 5 pm 
Location : [Sunny cafe & kitche](https://www.zomato.com/bangalore/sunny-cafe-kitchen-hsr) ,  HSR Layout 
@daviddias @KottackalJacob @atvanguard @HariharasudhanAS  Any suggestion

**Comment by KottackalJacob** [comment]
Sorry..not in town on the day.

Good Luck for the meeting.

**Comment by kishansagathiya** [comment]
Yo, IPFS Bangalore meetup is a now a thing! https://www.meetup.com/Decentralized-Web-ProtoSchool-Meetup/
Already looking for venues for first meetup.

**Comment by shwetaudacious** [comment]
I am interested to join in Mumbai meetup

**Comment by aamirams93** [comment]
> I'm currently in Delhi and would be interested in getting an IPFS community going. if anyone is interested in joining. I could also use some help in organizing it and getting a place to host it.

This is brilliant idea

**Comment by yozaam** [comment]
> I am interested to join in Mumbai meetup

Yes, me too!

Also, @rkapur123 if you are still working on the projects ( I checked the site but didnt get anything )where are you based I am very eager to talk

**Comment by harshit2000** [comment]
> I'm currently in Delhi and would be interested in getting an IPFS community going. if anyone is interested in joining. I could also use some help in organizing it and getting a place to host it.

Hi, I am mostly excited to host community meets and sessions. Feel free to drop in a mail to harshit2k2k@gmail.com
